includes/Checker/Checks/Abstract_PHP_CodeSniffer_Check.php

```diff
@@ -88,7 +89,7 @@ final public function run( Check_Result $result ) {
 		// Run PHPCS.
 		try {
 			ob_start();
-			$runner = new \PHP_CodeSniffer\Runner(); // phpcs:ignore SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName
+			$runner = new Runner();
```

Comment:
Should we also change `if ( ! class_exists( '\PHP_CodeSniffer\Runner' ) ) {` above?